src/plot/sample/gc_content_distribution.rs

```diff
@@ -78,8 +82,13 @@ impl SamplePlot for GCContentDistributionPlot {
         plot.add_trace(trace);
 
         // (6) Configure the graph for plotting and return.
+        let title = match sample {
+            Some(s) => format!("{} - {}", self.name(), s.clone()),
+            None => self.name().to_string(),
```

Comment:
Just my opinion, but I think it's more idiomatic to use `String::from(self.name())` here rather than `self.name().to_string()`. This is mainly because `String::from` comes uses the `std::convert::From` trait, which is commonly used for type conversions. `to_string` technically works, but it is tied to the `alloc::string::ToString` trait, which is much more tied to `Display`.